TypeCobol.Test/Parser/Copies/WithDiagnostics.rdzMix.txt

```diff
@@ -0,0 +1,14 @@
+Line -1[0,0] <37, Warning, General> - Warning: "END PROGRAM" is missing.
```

Comment:
I'm not sure about this diagnostic since we're parsing a copy and there's no 'PROGRAM-ID' either.

Reply:
Internally we are always parsing a program, the diagnostic here is created because the parsing is interrupted and the END PROGRAM is never reached. But I think this will change with #1897. I'm waiting for full validation of this PR before merging with latest devs on develop. I'll keep this conversation open until then.

Reply:
Ok let's see the result then. Ideally, the end-user should not messages relative to the wrapping program skeleton.